test/child_killing_test.rb

```diff
@@ -62,13 +62,13 @@ def assert_child_not_running(child_pid)
 
   if !defined?(RUBY_ENGINE) || RUBY_ENGINE != "jruby"
     it "old signal handling just kills off the child" do
-      worker_pid, child_pid, result = start_worker(0, false)
+      _worker_pid, child_pid, result = start_worker(0, false)
```

Comment:
At least in ruby 2.3.0 when run with the "-w" flag, unused variables with get a warning like: "lab.rb:2: warning: assigned but unused variable - bar".

The test suite seems to run with warnings enabled. Which seems sensible.